<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationJournal</class>
 <widget class="QWidget" name="ConfigurationJournal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Entry Settings and Range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ShowAllEntries">
        <property name="text">
         <string>Show &amp;all entries in the journal</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;b&gt;OR&lt;/b&gt; show entries from the last</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="YearRange">
            <item>
             <property name="text">
              <string>Year</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9 years</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 years</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="ShowRangeIndicator">
          <property name="text">
           <string>Show year range &amp;indicator on status bar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Select entry node icon</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="NodeIcon">
          <item>
           <property name="text">
            <string>Pencil #1 (original)</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pencil #2</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/pencil2.png</normaloff>:/icons/pencil2.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Small pencil</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/pencil-small.png</normaloff>:/icons/pencil-small.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Speech balloon</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/balloon-quotation.png</normaloff>:/icons/balloon-quotation.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Node</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/node.png</normaloff>:/icons/node.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Black bullets</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/bullet_black.png</normaloff>:/icons/bullet_black.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue bullets</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/bullet_blue.png</normaloff>:/icons/bullet_blue.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutuser">
     <property name="title">
      <string>User Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Your full name (as you would like it to appear):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FullName"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select your gender:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="Male">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use male avatar icon: &lt;img src=&quot;:/icons/user.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Male</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="Female">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use female avatar icon: &lt;img src=&quot;:/icons/user-female2.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Female</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Highlighting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="Highlight">
        <property name="toolTip">
         <string>&lt;p&gt;If selected, this option causes RoboJournal to &lt;span style=&quot;background: yellow; color: black&quot;&gt;highlight&lt;/span&gt; the current search term (with a color from the list below) wherever it appears in the text. Highlighting makes it &lt;i&gt;much&lt;/i&gt; easier to find instances of the current search term in the entry you are currently viewing.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Highlight current search term</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Select highlight color</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="HighlightColor">
          <item>
           <property name="text">
            <string>Yellow (default)</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight_yellow'.png</normaloff>:/icons/highlight_yellow'.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Orange</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight-orange.png</normaloff>:/icons/highlight-orange.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pink</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight-pink.png</normaloff>:/icons/highlight-pink.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Green</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight-green.png</normaloff>:/icons/highlight-green.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cyan</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight-cyan.png</normaloff>:/icons/highlight-cyan.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Purple</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/highlight-purple.png</normaloff>:/icons/highlight-purple.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="SortByDay">
        <property name="toolTip">
         <string>&lt;p&gt;This setting is recommended if you frequently write more than once per day because it conveniently groups all entries made on the same day. Default behavior is to sort entries by month instead.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Group journal entries by day </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ShowTitle">
        <property name="text">
         <string>Show &amp;title for current journal entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Autoload">
        <property name="text">
         <string>Display most &amp;recent entry automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UseFullName">
        <property name="toolTip">
         <string>If checked, RoboJournal displays your full name in the entry pane header.</string>
        </property>
        <property name="text">
         <string>Display full &amp;name (instead of username)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
